skimage/measure/_colocalization.py

```diff
@@ -0,0 +1,371 @@
+import numpy as np
+from scipy.stats import pearsonr
+from functools import wraps
+from .._shared.utils import check_shape_equality
+
+__all__ = ['pearson_r',
+           'manders_colocalization_coeff',
+           'manders_overlap_coeff',
+           'overlap',
+           'integrated_density',
+           'average_integrated_density'
+           ]
+
+
+def check_shape_equality_all(*args):
+    # check that all np.arrays passed to it have the same shape
+    im1 = args[0]
+    for im in args[1:]:
+        check_shape_equality(im1, im)
+    return
+
+
+def check_numpy_arr(arr, name, bool_expected=False):
+    if type(arr) != np.ndarray:
+        raise ValueError(
+            f"{name} is of type {type(arr)} not nd.array of dtype boolean as",
+            "expected")
```

Comment:
```suggestion
            f"{arr} is of type {type(arr)}; should be ndarray of dtype bool.",
```
I can't see the need for the `name` argument... (?)